### PR TITLE
[docs]: Django Docker Security

### PR DESCRIPTION
**Pull request by dmakeroam**
docs: improve security and image size

### CODE REVIEW

File: content/faq/docker/docker-context.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Docker context
+showMetadata: true
+editable: true
+showToc: true
+---
+
```
Comment (aaronamm) [inline content/faq/docker/docker-context.md:7]: Could you please a little introduction here to explain what is docker context?
Thanks.

File: content/web-frameworks/django/docker-compose-for-django-mysql.md [hunk 2]
```diff
@@ -59,18 +59,20 @@ services:
 
   db:
     container_name: mysql-server
-    image: mysql:latest
+    image: mysql:8.0.26
```
Comment (aaronamm) [inline content/web-frameworks/django/docker-compose-for-django-mysql.md]: @dmakeroam Same as previous question on MySQL docker, should it be mysql:8 for flexibility?

**Review by aaronamm — CHANGES_REQUESTED** [review]
@dmakeroam I have only a minor feedback.
It looks like your later PR is based on a previous PR which is not right for this situation.

Actually, you need to have each of your PR based on the main branch and keep only one changed file for each PR. 
This is because each of your PR targets to change only one docker-compose content.

Please let me know if you would like to reset this PR to have only one `docker-compose-for-django-mysql.md` file or not.
If not, I think we need to start merging from the first one then changes in other files will be removed from this PR automatically.

File: content/web-frameworks/django/docker-compose-for-django-mysql.md (hunk 2 above)
Comment (dmakeroam) [reply content/web-frameworks/django/docker-compose-for-django-mysql.md]: Yes

File: content/web-frameworks/django/docker-compose-for-django-mysql.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ To use Django and MySQL Docker compose, we need to create these required files a
 - Example content of `Dockerfile`
 ```sh
 # Dockerfile
-FROM python:3
+FROM python:alpine3.14
```
Comment (mildronize) [inline content/web-frameworks/django/docker-compose-for-django-mysql.md]: Here is optional comment, 

I think when we write python the developer need to know python version before writing code.
tag name of docker image should be self explanation for the reader.

in this case I found similar tag for `alpine3.14`, I think it should be `3.9-alpine3.14` that tell, "using python 3.9" for alpine 3.14

```
Ref From: https://hub.docker.com/_/python
3.9.6-alpine3.14, 3.9-alpine3.14, 3-alpine3.14, alpine3.14, 3.9.6-alpine, 3.9-alpine, 3-alpine, alpine
```

File: content/web-frameworks/django/docker-compose-for-django-mysql.md [hunk 4]
```diff
@@ -59,18 +59,20 @@ services:
...
-    image: mysql:latest
+    image: mysql:8.0
     restart: always
     environment:
-      # TODO move password to a secret file
-      MYSQL_ROOT_PASSWORD: MySQL123!
+      MYSQL_ROOT_PASSWORD_FILE: "/run/secrets/mysql_root_password"
       MYSQL_USER: my-user
-      MYSQL_PASSWORD: passw@rd
+      MYSQL_PASSWORD_FILE: "/run/secrets/mysql_password"
```
Comment (mildronize) [inline content/web-frameworks/django/docker-compose-for-django-mysql.md:67]: better way.

File: content/web-frameworks/django/docker-compose-for-django-mysql.md [hunk 5]
```diff
@@ -59,18 +59,20 @@ services:
...
       MYSQL_DATABASE: my-db
     ports:
       - 3306:3306
+    secrets:
+      - mysql_root_password
+      - mysql_password
     healthcheck:
-      test: mysqladmin ping -h localhost -u $$MYSQL_USER --password=$$MYSQL_PASSWORD
+      test: mysqladmin ping -h localhost -u $$MYSQL_USER --password=$$(cat /run/secrets/mysql_password)
```
Comment (mildronize) [inline content/web-frameworks/django/docker-compose-for-django-mysql.md:75]: Nice!

**Review by mildronize — COMMENTED** [review]
Thank you for your contribution, just minor optional feedback.

**Review by aaronamm — APPROVED** [review]

**Review by mildronize — APPROVED** [review]
Well Done, thank you for your change.

File: content/web-frameworks/django/docker-compose-for-django-mysql.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ To use Django and MySQL Docker compose, we need to create these required files a
 - Example content of `Dockerfile`
 ```sh
 # Dockerfile
-FROM python:3
+FROM python:3.9-alpine3.14
```
Comment (aaronamm) [inline content/web-frameworks/django/docker-compose-for-django-mysql.md:19]: Nice

**Pull request merged**